sqldb/migrations_prod.go

```diff
@@ -2,4 +2,10 @@
 
 package sqldb
 
-var migrationAdditions []MigrationConfig
+var migrationAdditions = []MigrationConfig{
+	{
+		Name:          "000007_invoice_add_settled_index",
+		Version:       8,
+		SchemaVersion: 7,
+	},
+}
```

Comment:
still not the correct place @ziggie1984 - this should go directly into the main line. 

This struct should always remain emtpy